lib/win32.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+const pify = require('pify');
+const fs = require('fs');
+const path = require('path');
+const readdir = pify(fs.readdir);
+const stat = pify(fs.stat);
+const writeFile = pify(fs.writeFile);
+const fswin = require('fswin');
+
+/**
+ * Modify options object as needed for Windows
+ * @param opts Object
+ * @returns Object
+ */
+const handleOptions = opts => {
+  // Convert any CLI params to boolean
+  Object.keys(opts).forEach((key) => {
+    const val = opts[key];
+    if (typeof val === 'string') {
+      opts[key] = opts[key] === 'true';
+    }
+  });
+
+  if (opts.clear) {
+    opts.dependencyHidden = opts.devDependencyHidden = false;
+  }
+  return opts;
+};
+
+/**
+ * This hides all the dependencies first and writes the info file
+ * @param projectPath String
+ * @param opts Object
+ * @returns {Promise}
+ */
+const performPreAction = (projectPath, opts) => {
+  const modules = path.join(projectPath, 'node_modules');
+  const hidden = typeof opts.clear === 'boolean' ? !opts.clear : true;
+
+  writeFile(
```

Comment:
Ok, well if you don't want to throw an error then I guess the original `console.warn` is the best approach.